packages/core-flows/src/user/workflows/create-users.ts

```diff
@@ -0,0 +1,15 @@
+import { CreateUserDTO, UserDTO } from "@medusajs/types"
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { createUsersStep } from "../steps"
+
+interface CreateUsersInput {
```

Comment:
q: This interfaces for me should be in the types package. Because this is a public facing API. As a user, I could be consuming a workflow and be able to type my input before passing it to the workflow. I think it should moves in general. wdyt? Also, when we will allow for extension the users might need to access this interface so it should be at least exported.

Same question for all workflows interfaces, we did start to put them in the types package from memories

cc @olivermrbl @carlos-r-l-rodrigues

Reply:
Yeah, we can start to put these in the types package. I don't have any strong opinion on whether we do it now or later. A larger clean-up of types is due, but that could all be tackled after 2.0rc.